packages/snaps-cli/src/cmds/init/initHandler.test.ts

```diff
@@ -115,15 +122,18 @@ describe('initialize', () => {
         .spyOn(initUtils, 'isInGitRepository')
         .mockImplementation(() => false);
       jest.spyOn(initUtils, 'gitInit').mockImplementation();
+
+      const { manifest, packageJson } = getSnapFiles({
+        packageJson: { main: undefined } as any,
```

Comment:
`@ts-expect-error`?

Reply:
Actually this should probably just be typed correctly, will fix.